changelog.d/586.misc

```diff
@@ -0,0 +1 @@
+Run "npm install" to fix missing js-yaml issue
```

Comment:
Probably would prefer something like `Regenerate package-lock.json to fix a build issue`

Reply:
```suggestion
Regenerate package-lock.json to fix a build issue
```